cordova-lib/spec-cordova/helpers.js

```diff
@@ -17,12 +17,23 @@
     under the License.
 */
 
-var path = require('path'),
-    fs = require('fs'),
-    shell = require('shelljs'),
-    os = require('os');
+var path         = require('path'),
+    fs           = require('fs'),
+    shell        = require('shelljs'),
+    os           = require('os'),
+    ConfigParser = require('../src/configparser/ConfigParser');
 
-module.exports.tmpDir = function(subdir) {
+// Just use Android everywhere; we're mocking out any calls to the `android` binary.
+module.exports.testPlatform = 'android';
+
+function getConfigPath(dir) {
+    if (dir.indexOf('config.xml') !== dir.length - 10) {
```

Comment:
@purplecabbage, I was leaning towards a comment at first, but then I realized we already have underscore which provides this functionality. I think we should just use it and I don't find it to be overkill.